frontend/pnpm-lock.yaml

```diff
@@ -4994,7 +4994,7 @@ packages:
       normalize-path: 3.0.0
       readdirp: 3.6.0
     optionalDependencies:
-      fsevents: 2.3.2
+      fsevents: 2.3.3
```

Comment:
pnpm lock update doesn't seem to be related to the change

Reply:
As intended,  it's to make `npx update-browserslist-db@latest` go away.